ext/datastore/bigquery/table_spec.go

```diff
@@ -145,7 +148,7 @@ func (s tableSpecHandler) ToYaml(optResource models.ResourceSpec) ([]byte, error
 func (s tableSpecHandler) FromYaml(b []byte) (models.ResourceSpec, error) {
 	var yamlResource TableResourceSpec
 	if err := yaml.Unmarshal(b, &yamlResource); err != nil {
-		return models.ResourceSpec{}, err
+		return models.ResourceSpec{}, nil
```

Comment:
Is this done by mistake?

Reply:
Oops yes it was done by mistake, thanks for pointing out